src/_includes/docs/install/reqs/linux/software.md

```diff
@@ -24,16 +24,14 @@ To develop Flutter on {{include.os}}:
 1. To develop {{include.target}} apps:
 
    {:type="a"}
-   1. Install the following prerequisite packages for Android Studio.
+   1. Install the following prerequisite packages for Android Studio:
 
       ```console
-      $ sudo apt-get install \
-          libc6:amd64 libstdc++6:amd64 \
-          libbz2-1.0:amd64 libncurses5:amd64
+      $ sudo apt-get install libc6:amd64 libstdc++6:amd64 lib32z1 libbz2-1.0:amd64
```

Comment:
`libstdc++6` could also be unnecessary

It's also no longer listed in Android Studio deb package https://github.com/flutter/website/issues/11252#issuecomment-2418427182

Reply:
Hmm thanks. These docs need some further updates, so I'll leave out dropping the dependency for follow up.